qiskit_braket_provider/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from .providers import (
     AWSBraketProvider,
-    AWSBraketJob,
```

Comment:
This might be a breaking change for code which does something like `from qiskit_braket_provider import AWSBraketJob`

Reply:
I would expect the developer to be importing `AmazonBraketTask` therefore `AWSBraketJob` is not needed in 

```"""Qiskit-Braket provider."""

from .providers import (
    AWSBraketProvider,
    AmazonBraketTask,
    AWSBraketBackend,
    BraketLocalBackend,
)```

or are you saying I should keep `AWSBRaketJob` as well?

Reply:
ah I understand what you mean now!

Reply:
this has now been resolved with the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/abb7925956ff8970d4f3399629be30011655ddb6